src/bench/bench_dash.cpp

```diff
@@ -45,7 +45,7 @@ int main(int argc, char** argv)
     SetupBenchArgs();
     std::string error;
     if (!gArgs.ParseParameters(argc, argv, error)) {
-        tfm::format(std::cerr, "Error parsing command line arguments: %s\n", error.c_str());
+        tfm::format(std::cerr, "Error parsing command line arguments: %s\n", error);
```

Comment:
17299 / 9adca5a4f9b1d02b825a11930d5f7a5862cd65be : `c_str` changes were squashed into the wrong commit it seems, should go to 17279 / f0f803bab26fc7776688e0fd3bec21ac9aab3b29 (same for similar cases below in this commit)